static/app/app/controller.js

```diff
@@ -57,6 +58,35 @@ var Controller = Marionette.Object.extend({
             spots.fetch().done(callback);
         }
 
+    },
+    join: function(moveId) {
+      var moves = channel.request('entities:moves');
+
+      $.when(moves.fetch()).done(function(){
+        var currentMove = channel.request("entities:move");
+        var moveToJoin = channel.request('entities:moves').find({id: parseInt(moveId)});
+
+        currentMove.set('time', moveToJoin.get('time').format());
+        currentMove.set('spot', moveToJoin.get('spot'));
+
+        if (currentMove.get('user')) {
+          currentMove.save({}, {
```

Comment:
It looks like the tabbing got screwed up here. We like to keep it a soft tabs with an indent of 4 spaces.